src/interlocking_component/router.py

```diff
@@ -0,0 +1,5 @@
+class Router:
+    def get_route(
+        start_track_segment_id: track_segment_id, end_track_segment_id: track_segment_id
+    ):
```

Comment:
```suggestion
    def get_route(
        self, start_track_segment_id: str, end_track_segment_id: str
    ):
```
You should add a reference to the router (`self`) and the types of the parameter should be python types (`str`, if these are SUMO-ids). Additionally, you should add documentation to the method.